sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.VisualBasic.nuspec

```diff
@@ -12,7 +12,8 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio and/or the SonarQube platform.</summary>
     <description>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio (http://vs.sonarlint.org) and/or the SonarQube platform (http://www.sonarqube.org).</description>
-    <releaseNotes>http://www.sonarlint.org/visualstudio/rules/index.html#version=6.6.0.0</releaseNotes>
+    <!-- The link to the release notes needs to be updated manually (the version part) -->
+    <releaseNotes>https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10112&amp;version=13300</releaseNotes>
```

Comment:
Make sure this manual step is described in release process (if not there already).